Week1 (#4)
bitprj/Bitcamp-Serverless: week1/homework/issues/github.md

## Learning GitHub

> *For this assignment, you will follow the instructions and at the end, complete a task showing your knowledge of the subject. If at any moment you need help, feel free to contact your TAs.*

Reviewer: ```suggestion
*For this assignment, you will follow the instructions and complete a task showing your knowledge of the subject at the end. If at any moment you need help, feel free to contact your TAs.*
```

### GitHub

> If you haven't heard of GitHub before, you must have been living under a rock for the past ten years. GitHub is a cloud-based repository hosting service that is widely used in the tech industry. It allows teams to use Git for version control, collaboration, and file management. If you don't know what Git is, it's a version-control system for tracking changes in source code during software development. It is designed for coordinating work among programmers. 

Reviewer: ```suggestion
If you haven't heard of GitHub before, you must have been living under a rock for the past few years. GitHub is a cloud-based repository hosting service that is widely used in the tech industry. It allows teams to use Git for version control, collaboration, and file management. If you don't know what Git is, it's a version-control system for tracking changes, managing state, and concurrently developing on the same files or directories. Git and Github's tools are specifically designed to make coordinating work easier, and they are one of the most pervasive shared tools among students and the industry.
```

If you want to learn more about *what it is* and *how to use it*, try taking [this](https://lab.github.com/githubtraining/introduction-to-github#:~:text=Audience.%201%20Assign%20yourself.%20Assign%20the%20first%20issue,an%20issue.%20%204%20Create%20a%20branch.%20) GitHub Learning Lab Course. After finishing it, you will have a strong understanding of all the features GitHub has to offer. To make an account, go to: https://github.com/join to sign up. After making an account, you're all set to complete Task 3!

> **Task 3: create a temporary repository and commit a readme.md file (make sure you know key functions like commits, forking, pull request, branch, etc).**

Reviewer: ```suggestion
**Task 3: create a repository and commit a README.md file (make sure you know key functions like commits, forking, pull request, branch, etc).**
```
